src/app/ui/components/account/account.card.tsx

```diff
@@ -89,9 +100,26 @@ export function AccountCard({
                 display="inline-block"
                 overflow="hidden"
               >
-                {balance}
+                {totalBalance}
               </PrivateTextLayout>
             </styled.h1>
+            <styled.h2 textStyle="label.02" color="ink.text-subdued" mt="space.01">
+              Available balance:
+              <styled.span ml="space.01">
+                <BasicTooltip
+                  side="right"
+                  label="Some funds may be unavailable to protect your collectible assets. Disable protection to access your remaining balance."
```

Comment:
Maybe a bit more specificity on where: "... You can ensure those fund are available by disabling protection in the collectible menu."